### PR TITLE
Tiny ruby styleguide fix

### PR DESCRIPTION
**Pull request by iamjarvo**

### CODE REVIEW

File: lib/quandl/api_config.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def api_base
       end
 
       def api_version=(api_version)
-        @api_version = api_base if @api_version.nil?
+        @api_version ||= api_base
```
Comment (mbasset) [inline lib/quandl/api_config.rb]: @iamjarvo could you also correct the variable name which seems to wrong here also?

**Comment by mbasset** [comment]
Thanks @iamjarvo would you be able to correct the comment above and the other rubocop changes.

File: lib/quandl/api_config.rb (hunk 1 above)
Comment (iamjarvo) [reply lib/quandl/api_config.rb]: Good catch! That was a silly mistake on my part. I tried to add a test for that error but the suite still passed. At some point I can take a look at that

**Comment by iamjarvo** [comment]
Do you support ruby 2.0?

**Comment by mbasset** [comment]
@iamjarvo do you mean the 2.x series or specifically ruby 2.0?

**Comment by iamjarvo** [comment]
2.0. I was wondering if 2.0 can be an allowed failure?

**Comment by mbasset** [comment]
Generally for this we would like to keep support for supported versions of ruby. That being said this post https://www.ruby-lang.org/en/news/2017/04/01/support-of-ruby-2-1-has-ended/ would indicate that we should drop support for ruby 2.0 and 2.1. If you would like to drop those from the travis.yml I can approve + merge the PR.

**Pull request merged**